lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -18,11 +18,11 @@ def self.provision_execute(_params = {}, _secrets = {}, context = {})
           object_type, object_id = context.execution.values_at("_object_type", "_object_id")
 
           # ensure we are in a provisioning request
-          return BuiltinRunner.error!(runner_context, :cause => "Calling provision_execute on non-provisioning request: [#{object_type}]") unless object_type == "ServiceTemplateProvisionTask"
-          return BuiltinRunner.error!(runner_context, :cause => "Missing MiqRequestTask id") if object_id.nil?
+          return ::ManageIQ::Providers::Workflows::BuiltinRunnner.error!({}, :cause => "Calling provision_execute on non-provisioning request: [#{object_type}]") unless object_type == "ServiceTemplateProvisionTask"
```

Comment:
This is pretty annoying, not even `module_parent::BuiltinRunner` works

Reply:
that's so weird - I'm sure it's a BasicObject + zeitwerk problem - maybe we need to manually require it ahead of time?